css/src/scss/components/_iconic-button.scss

```diff
@@ -0,0 +1,36 @@
+/* -----------------------------------------
+    ## Iconic Button
+----------------------------------------- */
+
+.IconicButton {
+  display: inline-flex;
+  border: none;
+  justify-content: center;
+  align-items: center;
+  outline: none;
+  cursor: pointer;
+  &:focus i {
+    box-shadow: $shadow-focus;
+  }
+  &:hover i {
+    background: $color-base-light-blue;
+  }
+  &:active i {
+    transform: scale(0.875);
+    transition: 0.2s ease-in;
+  }
+  i {
```

Comment:
we should probably give this a class in case people use other elements, eg svg or img, since it would still fit our naming convention. something like `IconicButton__Icon`